### PR TITLE
update the workload proposl

### PR DESCRIPTION
**Pull request by kwb0523**
**What type of PR is this?**
update the workload proposl

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: docs/proposal/kmesh_support_workload.md [hunk 1]
```diff
@@ -116,26 +120,21 @@ typedef struct
...
-
 ## Traffic governance process
 
 ![traffic_governance](./pics/traffic_governance.svg)
 
-* Client Access Service: Search the serviceinfo map based on the IP and Port accessed by the client, find the corresponding service_id, and then search the service map based on the service_id to find the endpoint_count of the backend Pod in the service. Then, search the endpoint map based on the service_id and the random backend_index generated based on the count to find the corresponding backend_uid. Finally, use the backenduid to find the IP and Port of the backend.
-* Client access Pod: Access directly through Pod's IP and Port.
+* traffic governance process: Search the frontend map based on the IP accessed by the client, find the corresponding upstream_id, and then use this upstream_id to search for the service map and backend map:
+if the corresponding service is found in service map, and get the endpoint_count of the backend Pod in the service. Then, search the endpoint map based on the service_id and the random backend_index generated based on the count to find the corresponding backend_uid. Finally, use the backenduid to find the IP and Port of the backend.
```
Comment (LiZhenCheng9527) [inline docs/proposal/kmesh_support_workload.md:136]:
```suggestion
If the corresponding service is found in service map, and get the endpoint_count of the backend Pod in the service. Then, search the endpoint map based on the service_id and the random backend_index generated based on the count to find the corresponding backend_uid. Finally, use the backenduid to find the IP and Port of the backend.
```

File: docs/proposal/kmesh_support_workload.md [hunk 2]
```diff
@@ -116,26 +120,21 @@ typedef struct
...
+if the corresponding backend is found in backend map, it is a directly pod access.
```
Comment (LiZhenCheng9527) [inline docs/proposal/kmesh_support_workload.md:137]: ditto

**Review by LiZhenCheng9527 — COMMENTED** [review]
I think it's better to centre the image
can refer to [center](https://blog.csdn.net/Jivove/article/details/123496699)

File: docs/proposal/kmesh_support_workload.md (hunk 1 above)
Comment (hzxuzhonghu) [inline docs/proposal/kmesh_support_workload.md:136]: we should seperate the steps with ordered list

**Review by hzxuzhonghu — APPROVED** [review]
/lgtm

**Pull request merged**